aiohttp/payload.py

```diff
@@ -150,6 +151,11 @@ def __init__(self, value, *args, **kwargs):
 
         self._size = len(value)
 
+        if self._size > DEFAULT_LIMIT:
```

Comment:
I think `DEFAULT_LIMIT` is too small. lets define new const, something close to 1mb